### PR TITLE
Added it_IT internationalisation.

### PR DESCRIPTION
**Pull request by gianarb**
This PR translates Zipkin in Italian.

### CODE REVIEW

**Comment by codefromthecrypt** [comment]
Thank you @gianarb! do you mind rebasing and taking a second look? @MrGlaucus fixed some style issues in his last change which might affect you.. maybe not

**Comment by MrGlaucus** [comment]
I added Service Name and Span Name labels in index.mustache, so you might have to add two more translations in i18n file, thank you so much.

**Comment by gianarb** [comment]
All done!

**Comment by codefromthecrypt** [comment]
@drolando ptal

File: zipkin-ui/static/traces_it_IT.properties [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+duration-desc = Piú lunghi prima
+duration-asc = Piú corti prima
+timestamp-desc = Nuovi prima
+timestamp-asc = Vecchi prima
+traces.des = Seleziona i criteri per ricercare le tracce.
+modal.title = Come trovare una traccia
+modal.p1 = Stai cercando una traccia eh? Sei nei posto giusto!
+modal.title2 = Nome servizi
+modal.p2 = Prima di tutto seleziona un sevizio per cui cercare. Tutte le tracce sono indicizzate per nome del servizio.
+modal.p3 = Once that is done you have three options to pick from. See notes on each below.
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]: duplicated string, delete

File: zipkin-ui/static/traces_it_IT.properties [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+modal.p3 = Quando hai fatto hai tre opzioni tra cui scegliere. Guarda le note per ognuna di loro sotto.
+modal.title3 = 1. Ricerca per nome dello span
+modal.p4 = Span sono generalmente metodi thrift o endpoints Rails. Questo ti permette di recuperare tracce che accedono ad una particolare parte del servizio.
+modal.title4 = 2. Ricerca per annotazione
+modal.p5 = Un annotazione é un punto nel tempo all'iterno della traccia. é un valore di tipo stringa, questo e quello che puoi utilizzare come parametro di ricerca. Possono essere come tipo
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]: - Un'annotazione
- iterno --> interno

I'd say: "Il suo valore e' di tipo stringa ed e' quello che puoi utilizzare come parametro di ricerca. Possono essere cose come"

File: zipkin-ui/static/traces_it_IT.properties [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+modal.title5 = 3. Ricerca per chiave/valore
+modal.p6 = Annotazione di tipo chiave / valore sono un metadato extra attaccato ad un trace. Per esempio l'url chiamato, il codice di risposta o un eccezione. Guarda sotto per alcuni esempio.
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]: - Annotazioni
- una traccia
- esempi
- un'eccezione

File: zipkin-ui/static/traces_it_IT.properties [hunk 4]
```diff
@@ -0,0 +1,29 @@
+traces.servicename = Nome del servizio
+traces.spanname = Nome dello Span
+traces.stime = Inizio
+traces.etime = Fine
+traces.duration = Durata (μs) >=
+traces.limit = Limite
+traces.findBtn = Trova tracce
+traces.sort = Ordina
+service-percentage-desc = Service Percentage: Longest First
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]: translation missing

File: zipkin-ui/static/traces_it_IT.properties [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+service-percentage-asc = Service Percentage: Shortest First
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]: translation missing

File: zipkin-ui/static/traces_it_IT.properties [hunk 6]
```diff
@@ -0,0 +1,29 @@
+traces.servicename = Nome del servizio
+traces.spanname = Nome dello Span
```
Comment (drolando) [inline zipkin-ui/static/traces_it_IT.properties]: span lowercase

File: zipkin-ui/static/trace_it_IT.properties [hunk 7]
```diff
@@ -0,0 +1,7 @@
+trace.duration = Durata:
+trace.service = Servizi:
+trace.depth = Profinditá:
+trace.spans = Spans totali:
```
Comment (drolando) [inline zipkin-ui/static/trace_it_IT.properties]: nit: you usually don't add the plural "s" when using English words in Italian. I'd say "Span totali:"

File: zipkin-ui/static/traces_it_IT.properties (hunk 2 above)
Comment (drolando) [reply zipkin-ui/static/traces_it_IT.properties]: Btw, I know that the English version is the same, but is it normal that the sentence doesn't end?

**Review by drolando — CHANGES_REQUESTED** [review]
Mostly looks good to me, just a few things to fix

File: zipkin-ui/static/traces_it_IT.properties (hunk 2 above)
Comment (codefromthecrypt) [reply zipkin-ui/static/traces_it_IT.properties]: "<strong>finagle.timeout</strong>" follows this, so yeah abnormal.. maybe comment deserving? or perhaps better to scrap the strong and inline the annotation it refers to

**Comment by gianarb** [comment]
Thanks @drolando !

**Comment by codefromthecrypt** [comment]
Thank, you two!

**Pull request merged**